packages/shared/src/hooks/useFeed.ts

```diff
@@ -213,15 +213,16 @@ export default function useFeed<T>(
       );
     }
     return newItems;
-    // @NOTE see https://dailydotdev.atlassian.net/l/cp/dK9h1zoM
-    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Just to make sure that for some reason the right dependencies were not set, so let's just keep a good eye when testing

Reply:
Yes, thankfully this will go through internal testing so we should have ample amount of time for testing it.